pkgs/tools/misc/nagstamon/default.nix

```diff
@@ -1,21 +1,26 @@
-{ lib, fetchurl, pythonPackages }:
+{ lib, fetchurl, pythonPackages, qt6 }:
 
 pythonPackages.buildPythonApplication rec {
   pname = "nagstamon";
-  version = "3.14.0";
+  version = "3.16.0";
 
   src = fetchurl {
     url = "https://github.com/HenriWahl/Nagstamon/archive/refs/tags/v${version}.tar.gz";
-    sha256 = "sha256-9RxQ/rfvoyjSUsY4tmAkBdVQqZYi3X6PBzQYFIeenzA=";
+    sha256 = "sha256-zqa2/1mkh6Cv7TguQiNq9AGlTa7mHxlpBfgCBpoKU5Y=";
   };
 
   # Test assumes darwin
   doCheck = false;
 
+  buildInputs = [ qt6.qtmultimedia qt6.qtbase qt6.qtsvg ];
+
   build-system = with pythonPackages; [ setuptools ];
-  dependencies = with pythonPackages; [ configparser pyqt6 psutil requests
+  dependencies = with pythonPackages; [ arrow configparser pyqt6 psutil requests
      beautifulsoup4 keyring requests-kerberos lxml dbus-python python-dateutil pysocks ];
 
+  # Use wrapQtAppsHook to wrap the app properly
+  nativeBuildInputs = [ qt6.wrapQtAppsHook ];
+
   meta = with lib; {
```

Comment:
Set platforms = linux. Doesn't build on Mac.

Reply:
Platforms should represent what upstream supports. If it doesn't build we should mark it as a badPlatforms. The error looks like it just might need tweaking to fix. 

'''bash

    >               ^^^^^^^^^^^^^^^^^
       > NameError: name 'bdist_rpm_options' is not defined. Did you mean: 'bdist_mac_options'?

Reply:
Sorry but I don't quite understand what you expect from me.
Sorry the goal of this PR is to fix this application which has been broken for several versions of nixos.

Reply:
Need to add just : platforms = platforms.linux; ?